src/ia-account-settings.ts

```diff
@@ -74,26 +74,26 @@ export class IAAccountSettings
   /**
    * contain profile picture URL
    *
-   * @type {String}
+   * @type {string}
    * @memberof IAUXAccountSettings
    */
-  @property({ type: String }) profilePicture: String = '';
+  @property({ type: String }) profilePicture: string = '';
 
   /**
    * contain boolean status loan history visibility
    *
-   * @type {String | Boolean}
+   * @type {string | boolean}
    * @memberof IAUXAccountSettings
    */
-  @property({ type: String }) loanHistoryFlag: String | Boolean = '';
+  @property({ type: String }) loanHistoryFlag: string | boolean = '';
```

Comment:
No changes for this PR, but was just curious if you knew what the states were that would cause this to be a boolean or a string?